### PR TITLE
164796853 Complete - Users Should Be Able To Create Articles

### PR DESCRIPTION
**Pull request by chikeozulumba**
#### Description
Complete the feature: Articles form the backbone of our application and introduce us to **CRUD** which is the building block of most web applications.

_**As a User**_
I want to be able to create my articles and read/access articles created by other authors. 
I want to be able to **ONLY** update and delete articles that i have authored.
Let's get creative in the design of this.

#### Type of change
- [ ] Bug Fix (a non-breaking change which fixes an issue)
- [x] New Feature (a non-breaking change that adds functionality)
- [ ] New Chore (a non-breaking change that adds functionality)
- [ ] Breaking Change (a fix or feature which would cause existing functionality not to work as expected)
- [ ] This change requires a documentation update
#### How Has This Been Tested?
- [ ] End to End
- [ ] Integration
#### Checklist:
- [ ] Unit test
- [ ] Code coverage
#### Linked to PR 
[164796853 Users Should Be Able To Create Articles](https://github.com/andela/cosmos-ah-frontend/pull/29)
#### PT
[#164796853](https://www.pivotaltracker.com/story/show/164796853)

### CODE REVIEW

File: src/components/ArticleComponent/Create/Content/Tags/index.jsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
+/* eslint-disable no-undef */
```
Comment (Oluwaseyi000) [inline src/components/ArticleComponent/Create/Content/Tags/index.jsx:1]: do you mind explaining why you disable eslint no-undef for this page?

**Review by Oluwaseyi000 — COMMENTED** [review]
great work, thumb up! on a lighter note, I feel your create-article state can share the same state folder with existing article folder 
Screenshot 2019-05-16 at 3 12 05 PM

File: src/lib/colors.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
+export const colors = [
```
Comment (Raymond-Osy) [inline src/lib/colors.js:1]: Nice idea with these colors. However, I feel the majority of the colors in this array would not be used or required and some individuals might just call in the colors they need than referring to this file to pick a color, @chikeozulumba what do you think? any clarifications?

Comment (Raymond-Osy) [reply src/lib/colors.js:1]: @chikeozulumba you might also want to reduce the file changes...good job!

**Review by Oluwaseyi000 — DISMISSED** [review]

File: src/lib/colors.js (hunk 2 above)
Comment (chikeozulumba) [reply src/lib/colors.js:1]: The colors are a util method for randomizing background colors on input - tags specifically.
The user won't in any way interact with it

Comment (chikeozulumba) [reply src/lib/colors.js:1]: > @chikeozulumba you might also want to reduce the file changes...good job!

I removed a lot of unnecessary files hence the files changes.

**Review by Temmyogunbo — APPROVED** [review]

**Pull request merged**